### PR TITLE
creating another branch for practice

### PR DESCRIPTION
**Pull request by VDozorov**
this is for practice

### CODE REVIEW

**Comment by amautacelt** [comment]
add console.log

**Review by amautacelt — CHANGES_REQUESTED** [review]
practice should be praktice

**Review by VDozorov — COMMENTED** [review]
Ok I will change that in the next upload

**Review by MHasan159 — CHANGES_REQUESTED** [review]
add another console log

File: demo.txt [hunk 1]
```diff
@@ -1 +1 @@
-Goodbye
\ No newline at end of file
+Practice for pull requests
```
Comment (amautacelt) [inline demo.txt]: change "pull" to "pulling"

File: demo.txt [hunk 2]
```diff
@@ -1 +1 @@
-Goodbye
```
Comment (MHasan159) [inline demo.txt:1]: Well Done

**Review by amautacelt — CHANGES_REQUESTED** [review]
reviewing

File: demo.txt (hunk 1 above)
Comment (VDozorov) [reply demo.txt]: Ok I will change that in the next upload

**Review by MHasan159 — APPROVED** [review]

**Review by amautacelt — APPROVED** [review]
Fantastic!

**Comment by VDozorov** [comment]
Merging

**Pull request merged**